testing/containers/ceph/Dockerfile

```diff
@@ -38,7 +38,6 @@ ENV PATH="${PATH}:/opt/go/bin"
 ENV GOROOT=/opt/go
 ENV GO111MODULE=on
 ENV GOPATH /go
-ENV GODEBUG=cgocheck=2
```

Comment:
May I ask why are we removing the above(maximum) pointer check?

Reply:
It doesn't exist anymore in Go 1.21.

Reply:
Ah..I see your `Pinner` type mentioned in the runtime section 😊